### ISSUE TITLE
[33] Obrazovka s textem - směry psaní

### ISSUE THREAD

**Issue by motyc** [LIBCAS/INDIHU-Exhibition#372]
Obrazovka s textem: Možnost psát text i po jiných osách (diagonálně, vertikálně).

**Comment by nnis** [comment]
je třeba připravit podklady pro funkcionalitu, ideálně @H-a-n-k-a , je možné se inspirovat grafickou poezií. na druhou stranu mě napadá, že s AI by bylo možné si takto vytvořit obrázek a místo obrazovky s textem dát obrazovku s obrázkem. Vyzkoušela jsem to s pár větami a první zadání bylo udělat to jako strom, druhý jako hvězdu, kde by byly ty diagonály. v tom případě by to šlo přidat jako inspiraci do manuálu. Navrhovala bych sekci **Využití AI ve výstavách** a šlo by tam toto, nebo nahráván audia

![Image](https://github.com/user-attachments/assets/5145c380-bf4c-4089-87d7-77f71a42c96b)
![Image](https://github.com/user-attachments/assets/57e594fc-cce9-4178-888e-18b3253e6808)

**Comment by H-a-n-k-a** [comment]
Ano, souhlasím, že díky současným možnostem pomoci AI již není tato funkcionalita přímo potřeba. Připravíme sekci kreativní práce s AI do manuálu?
